### PR TITLE
Add OpenAPI to glossary

### PR DESCRIPTION
**Pull request by fakela**

### CODE REVIEW

File: get-started/mindee-glossary.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ This glossary introduces common Mindee terminology. You will frequently encounte
 - **Documents**: This includes semi-structured documents such as an invoice, receipt, ID document, W9-forms, train-ticket etc in a PDF or image format.
 - **Labels**: A label key is a list containing valid JSON objects whose format depends on the type of feature you are annotating. 
 - **Models**: Models are machine learning files that can be trained to carry out specific tasks such as extracting the total amount info from receipts etc.
+- **OpenAPI Definition**: This is the standard format to define and structure REST APIs.
```
Comment (fharper) [inline get-started/mindee-glossary.md:17]: This is a specification that defines a standard, language-agnostic interface to RESTful APIs.

**Review by fharper — CHANGES_REQUESTED** [review]

File: get-started/mindee-glossary.md (hunk 1 above)
Comment (fakela) [reply get-started/mindee-glossary.md:17]: This suggestion is nice but seems too ambiguous for users not really great in English reading this.

Comment (fharper) [reply get-started/mindee-glossary.md:17]: But it's the definition of OpenAPI, what is ambigous? I feel like the previous sentence wasn't totally clear about the definition of OpenAPI...

**Review by fharper — APPROVED** [review]
LGTM

**Pull request merged**